spec/features/sends_rma_document_spec.rb

```diff
@@ -1,7 +1,12 @@
 require 'spec_helper'
 
 RSpec.describe 'Sending RMA Documents', :vcr do
-  let(:rma) { decorate(create(:return_authorization))  }
+  let(:rma) { decorate(create(:return_authorization, number: 'RA368041525'))  }
```

Comment:
Would these values make more sense in the `RMADecoratorStub`, since they aren't specific to this spec? I tend to prefer that strings or numbers in specs have expectations set against them, rather than just being part of the test setup. Meanwhile, the `RMADecoratorStub` exists precisely so the `rma` returns the "correct" values.